app/Controllers/CardsController.ts

```
@@ -0,0 +1,38 @@
+import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import Card from 'App/Models/Card'
+
+export default class CardsController {
+  public async index({}: HttpContextContract) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)